plugins/digitalocean/personal_access_token.go

```diff
@@ -32,6 +34,33 @@ func PersonalAccessToken() schema.CredentialType {
 		DefaultProvisioner: provision.EnvVars(map[string]sdk.FieldName{
 			"DIGITALOCEAN_ACCESS_TOKEN": fieldname.Token,
 		}),
-		Importer: importer.TryAllEnvVars(fieldname.Token, "DIGITALOCEAN_ACCESS_TOKEN"),
+		Importer: importer.TryAll(
+			importer.TryAllEnvVars(fieldname.Token, "DIGITALOCEAN_ACCESS_TOKEN"),
+			importer.MacOnly(TryDigitalOceanConfigFile("~/Library/Application Support/doctl/config.yaml")),
```

Comment:
Should we use the following indentation pattern for these for readability purposes?
```suggestion
			importer.MacOnly(
				TryDigitalOceanConfigFile("~/Library/Application Support/doctl/config.yaml")
			),
```